pkg/cmd/registry/artifact/util/pretty_print_references.go

```diff
@@ -0,0 +1,38 @@
+package util
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	registryinstanceclient "github.com/redhat-developer/app-services-sdk-core/app-services-sdk-go/registryinstance/apiv1internal/client"
+	"io"
+)
+
+type referenceRow struct {
+	Name       string `json:"name" header:"Reference Name"`
+	GroupId    string `json:"group" header:"Group"`
+	ArtifactId string `json:"artifactId" header:"Artifact ID"`
+	Version    string `json:"version" header:"Version"`
+}
+
+func PrettyPrintReferences(out io.Writer, references []registryinstanceclient.ArtifactReference) {
```

Comment:
I would prefer if there was some output when the artifact has not any reference. Now it prints two empty lines which in my opinion is confusing.